evmlab/tools/statetests/rndval/codesmart.py

```diff
@@ -111,11 +129,27 @@ def create_push_for_data(data):
                     yield create_push_for_data(self.randomSmallMemoryLength())
                     yield create_push_for_data(self.randomSmallMemoryLength())
                     args_filled = True
+                elif instr.name=="CREATE2":
+                    # todo: rework
+                    yield create_push_for_data(self.randomUniInt())
+                    yield create_push_for_data(self.randomSmallMemoryLength())
+                    yield create_push_for_data(self.randomSmallMemoryLength())
+                    yield create_push_for_data(self.randomUniInt())
```

Comment:
This is `value`, is randomUniInt small enough that we have a maybe 50% change of success do you think?